### ISSUE TITLE
Fix accessibility errors flagged by pa11y

### ISSUE THREAD

**Issue by kimadactyl** [geeksforsocialchange/gfsc-v3#210]
Suggestion: `npx pa11y https://gfsc.studio/`

- Check svg alt text for footer
- Check colour contrast

```
Results for URL: https://gfsc.studio/

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1:1. Recommendation:  change background to #702800.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
   ├── html > body > div > header > div > h2 > a > div:nth-child(1)
   └── <div class="header__h2">Geeks for Social Change</div>

 • Error: This emailinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputEmail.Name
   ├── #newsletter > form > input
   └── <input class="signupbig__input" type="email" placeholder="Your email address" name="fields[email]">

 • Error: Anchor element found with a valid href attribute, but no link content has been supplied.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
   ├── html > body > div > footer > div > div:nth-child(3) > div:nth-child(1) > a
   └── <a href="mailto:kim@gfsc.studio"> <svg xmlns="http://www...</a>

 • Error: Anchor element found with no link content and no name and/or ID attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.EmptyNoId
   ├── html > body > div > footer > div > div:nth-child(3) > div:nth-child(2) > a
   └── <a href=""> <svg id="Group_388" da...</a>

 • Error: Anchor element found with no link content and no name and/or ID attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.EmptyNoId
   ├── html > body > div > footer > div > div:nth-child(3) > div:nth-child(3) > a
   └── <a href=""> <svg id="Group_389" da...</a>

 • Error: Anchor element found with no link content and no name and/or ID attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.EmptyNoId
   ├── html > body > div > footer > div > div:nth-child(3) > div:nth-child(4) > a
   └── <a href=""> <svg id="Group_390" da...</a>

 • Error: Duplicate id attribute value "clip-path" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #clip-path
   └── <clipPath id="clip-path"> <rect id="Rectan...</clipPath>

 • Error: Anchor element found with a valid href attribute, but no link content has been supplied.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
   ├── html > body > div > footer > div > div:nth-child(3) > div:nth-child(5) > a
   └── <a href="https://twitter.com/gfscstudio"> <svg xmlns="http://www...</a>

 • Error: Anchor element found with a valid href attribute, but no link content has been supplied.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
   ├── html > body > div > footer > div > div:nth-child(3) > div:nth-child(6) > a
   └── <a href="https://social.gfsc.studio/@gfsc"> <svg xmlns="http://www.w3.org/...</a>

9 Errors
```

**Comment by katjam** [comment]
I think I made a checklist for Ivan when we did the sweep on Trans Dim which might also be a helpful starting point.

**Comment by katjam** [comment]
Let me know if you can't find it. I definitely have one that I use so I can paste in here (or a follow-up issue if we want to narrow the scope of this one).

**Comment by kimadactyl** [comment]
Yeah let's keep this ticket to just the output of this tool - a new ticket to do the audit sounds good, trying to keep this stuff a bit crispier
